github_connector/readme/ROADMAP.rst

```diff
@@ -0,0 +1,28 @@
+* For the time being, Github API doesn't provide some informations that are
```

Comment:
Some of these points were implemented already, isn't it?

Reply:
This first statment maybe is referring to the old tree view existing in previous versions of odoo (like v8) that displayed the elements in a hierarchical way